project/src/components/offer-card-component.tsx

```diff
@@ -0,0 +1,58 @@
+const AMOUNT_OFFER_CARD = 5;
+
+
+function offerCardRender() {
+  const arrayOfferCards = [];
+
+  for (let i = 0; i < AMOUNT_OFFER_CARD; i++) {
+    arrayOfferCards.push(OfferCardComponent({key: i}));
+  }
+
+  const offerCard = arrayOfferCards.map((OneOfferCard, index) =>
+    <div key ={OneOfferCard.id}>{OneOfferCard.content}</div>);
+
+  return offerCard;
+}
```

Comment:
убирай это отсюда, и переноси внутрь компонента MainPage, здесь должен остаться только компонент карточки

Reply:
а вообще это можно по короче переписать, причём прямо в jsx , например как то так
```suggestion
Array.from({ length: 5 }, (v, k) => k).map((el) =>  <OfferCardComponent key={el} />);

```

Reply:
плюс папку для этого компонента создай